acrobat/scripts/contentSecurityPolicy/dev.js

```diff
@@ -58,6 +58,26 @@ const connectSrc = [
   '*.hlx.live',
   'cdn.linkedin.oribi.io',
   'adobedc.demdex.net/',
+  '345-tti-184.mktoutil.com',
+  'conversions-config.reddit.com',
+  'srm.bf.contentsquare.net',
+  'www.facebook.com',
+  'www.google.com',
+  'adservice.google.com',
+  'analytics.pangle-ads.com',
+  'px.ads.linkedin.com',
+  'us-central1-adaptive-growth.cloudfunctions.net',
+  '345-tti-184.mktoresp.com',
+  'accounts.google.com',
+  'api.echosign.com',
+  'c.clicktale.net',
+  'ff.kes.v2.scr.kaspersky-labs.com',
+  'ff.kis.v2.scr.kaspersky-labs.com',
+  'files.acrobat.com',
+  'gc.kes.v2.scr.kaspersky-labs.com',
+  'gc.kis.v2.scr.kaspersky-labs.com',
+  'me.kes.v2.scr.kaspersky-labs.com',
+  'me.kis.v2.scr.kaspersky-labs.com',
```

Comment:
not sure about all of the changes, but at least the kaspersky-labs ones are a hard no.  if we're just going by logged CSP failures, some of them (many of them) may be blocked for good reason.